elastic-search-support/src/test/java/uk/gov/hmcts/ccd/definition/store/elastic/integration/CaseMappingGenerationIT.java

```diff
@@ -94,11 +94,13 @@ private CaseTypeEntity createCaseType() {
         CaseFieldEntity complexOfComplex = newComplexFieldOfComplex();
         CaseFieldEntity complexOfCollection = newComplexFieldOfCollection();
         CaseFieldEntity collectionOfBaseType = newCollectionFieldOfBaseType();
+        CaseFieldEntity dynamicField = newField("dynamicList", "DynamicList").build();
```

Comment:
```suggestion
        CaseFieldEntity dynamicListField = newField("dynamicList", FieldTypeUtils.BASE_DYNAMIC_LIST).build();
```

Reply:
Updated file and getting "DynamicList" from FieldTypeUtils.